CodeEdit/Features/CEWorkspace/Views/EnvironmentVariableListItem.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  EnvironmentVariableListItem.swift
+//  CodeEdit
+//
+//  Created by Axel Martinez on 9/4/24.
+//
+
+import SwiftUI
+
+struct EnvironmentVariableListItem: View {
+    @Binding var item: CETask.EnvironmentVariable
+    @Binding var selectedItemId: UUID?
+
+    var delete: (UUID) -> Void
+
+    /// State variables added to prevent an exception when deleting
+    /// the item in the onChange event
+    @State var name: String
+    @State var value: String
+
+    @FocusState private var isKeyFocused: Bool
```

Comment:
```suggestion
    @FocusState private var isKeyFocused: Bool
    
    @Binding var item: CETask.EnvironmentVariable
    @Binding var selectedItemId: UUID?

    /// State variables added to prevent an exception when deleting
    /// the item in the onChange event
    @State var name: String
    @State var value: String
    
    var delete: (UUID) -> Void
```